docs/api/rest_api/rest_api_reference/input/examples/ai/actions/GET/ActionConfiguration.json.example

```diff
@@ -0,0 +1,25 @@
+{
+    "ActionConfiguration": {
```

Comment:
I would regenerate this response as there were some changes to names, specifically lowercase attribute names